src/constants.rs

```diff
@@ -1,5 +1,7 @@
 pub const ALLOW_LIST: &[&'static str] = &[r".*\.xml$"];
 
+pub const EXIT_SUCCESS: i32 = 0;
+pub const EXIT_FAILURE: i32 = 1;
```

Comment:
Ran into a bunch of typing issues using std::ExitCode directly. It's typed as `u8`, but `ExitStatus` (which is returned by running the Command) is `i32`. I opted to define these since it reduces any concern of precision loss when converting from one form to another.

Reply:
I think this is fine. You can also alias it for better clarity. e.g. `pub type TrunkExitCode = i32`.